helm/cluster-autoscaler-app/templates/deployment.yaml

```diff
@@ -106,4 +110,8 @@ spec:
       volumes:
         - name: ssl-certs
           hostPath:
+            {{- if ((.Values.node).caBundlePath) }}
```

Comment:
```suggestion
            {{- if .Values.node.caBundlePath }}
```

Reply:
Removing parentheses would cause the templating to fail if .Values.node is not defined.

https://github.com/helm/helm/issues/8026
https://stackoverflow.com/questions/59795596/how-to-make-nested-variables-optional-in-helm/68807258#68807258